conditions/tests/tests.py

```diff
@@ -111,3 +116,17 @@ def test_nested_condlists(self):
         # Even though they both use either Gmail or Yahoo!, only Mr. X is targeted
         self.assertTrue(eval_conditions(self.long_term_gmail_yahoo_user_campaign, 'conditions', self.mrx))
         self.assertFalse(eval_conditions(self.long_term_gmail_yahoo_user_campaign, 'conditions', self.mrsy))
+
+
+class TestAdmin(DjangoConditionsTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.admin_user = User.objects.create_superuser("admin")
```

Comment:
Codacy found an issue: [No value for argument 'email' in method call](https://app.codacy.com/manual/RevolutionTech/django-conditions/pullRequest?prid=6342204)